01-shapefile-attributes.Rmd

```diff
@@ -42,6 +42,7 @@ to complete this lesson.
 
 
 * **raster:** `install.packages("raster")`
+* **sp:** `install.packages("sp")`
```

Comment:
sp should install with raster but it's good to call it out. my guess is we don't use ggplot in this lesson do we?